### ISSUE TITLE
Module error `svelte-meta-tags` when trying to implement

### ISSUE THREAD

**Issue by bartvdbraak** [subhendupsingh/sveltekit-notion-blog#1]
When trying to implement everything as per your documentation, I seem to be getting the following error:

```
3:38:47 AM [vite] page reload src/routes/+layout.svelte
3:38:48 AM [vite] Error when evaluating SSR module /src/routes/+layout.svelte: failed to import "sveltekit-notion-blog"
|- Error: Cannot find module 'svelte-meta-tags' imported from 'sveltekit-notion-blog/dist/components/BlogPost.svelte'
    at nodeImport (file:///<YOUR_PROJECT_PATH>/node_modules/vite/dist/node/chunks/dep-abb4f102.js:55911:25)
    at ssrImport (file:///<YOUR_PROJECT_PATH>/node_modules/vite/dist/node/chunks/dep-abb4f102.js:55813:30)
    at eval (<YOUR_PROJECT_PATH>/node_modules/sveltekit-notion-blog/dist/components/BlogPost.svelte:4:37)
    at async instantiateModule (file:///<YOUR_PROJECT_PATH>/node_modules/vite/dist/node/chunks/dep-abb4f102.js:55875:9)

Internal server error: Cannot find module 'svelte-meta-tags' imported from 'sveltekit-notion-blog/dist/components/BlogPost.svelte'
      at nodeImport (file:///<YOUR_PROJECT_PATH>/node_modules/vite/dist/node/chunks/dep-abb4f102.js:55911:25)
      at ssrImport (file:///<YOUR_PROJECT_PATH>/node_modules/vite/dist/node/chunks/dep-abb4f102.js:55813:30)
      at eval (<YOUR_PROJECT_PATH>/node_modules/sveltekit-notion-blog/dist/components/BlogPost.svelte:4:37)
      at async instantiateModule (file:///<YOUR_PROJECT_PATH>/node_modules/vite/dist/node/chunks/dep-abb4f102.js:55875:9)
Error: Cannot find module 'svelte-meta-tags' imported from 'sveltekit-notion-blog/dist/components/BlogPost.svelte'
    at nodeImport (file:///<YOUR_PROJECT_PATH>/node_modules/vite/dist/node/chunks/dep-abb4f102.js:55911:25)
    at ssrImport (file:///<YOUR_PROJECT_PATH>/node_modules/vite/dist/node/chunks/dep-abb4f102.js:55813:30)
    at eval (<YOUR_PROJECT_PATH>/node_modules/sveltekit-notion-blog/dist/components/BlogPost.svelte:4:37)
    at async instantiateModule (file:///<YOUR_PROJECT_PATH>/node_modules/vite/dist/node/chunks/dep-abb4f102.js:55875:9) {
  code: 'ERR_MODULE_NOT_FOUND'
}
```

You can view the code I currently have to implement this at
- https://github.com/bartvdbraak/hellob.art/tree/feat/blog
- https://github.com/bartvdbraak/hellob.art/pull/88

**Comment by bartvdbraak** [comment]
After installing `svelte-meta-tags` manually, it seems to get rid of the error.

**Comment by subhendupsingh** [comment]
Fixed the issues with commit 7170c77
